link_shortener/templates/base.html

```diff
@@ -158,10 +193,12 @@
           var x = shortLink.value;
           printout.innerHTML = "www.fueled.by/" + x;
         });
-      document.getElementById("shortlinked").style.color = "#AAA9BC";
-      document.getElementById("shortlinked").style.fontWeight = "bold";
+
+      if ($("div").is(".editAndCreate")) {
```

Comment:
This seems inefficient - it is picking all divs and looks for the editAndCreate class. This seems like a better solution - https://mkyong.com/jquery/how-to-check-if-an-element-is-exists-in-jquery/#:~:text=In%20jQuery%2C%20you%20can%20use,number%20of%20the%20matched%20elements.&text=To%20check%20if%20an%20element%20which,id%20of%20%E2%80%9Cdiv1%E2%80%9D%20exists.